src/app/components/Account/index.tsx

```diff
@@ -70,7 +70,7 @@ export const Account: FC<AccountProps> = ({ account, roseFiatValue }) => {
       )}
       <dt>{t('account.tokens')}</dt>
       <dd>
-        <TokenPills tokens={account.runtime_evm_balance} />
+        <TokenPills tokens={account.runtime_evm_balances || []} />
```

Comment:
This should be non-nullable when fetching a single account

Reply:
> This should be non-nullable when fetching a single account

Indeed; the API docs says this:

> NOTE 1: This field is omitted for efficiency when listing multiple accounts.

So I added a wrapper that adjusts the return type of the single account query, so that we will always know that this can never be null.